Makefile

```diff
@@ -1,6 +1,96 @@
-VERSION := $(shell python3 setup.py --version)
-NEXT_VERSION := $(shell expr "$(VERSION)" + 1)
-COMMIT=$(shell git rev-parse HEAD)
+#
+# Maintenance Helpers
+#
+# This makefile contains targets used for development, as well as helpers to
+# aid automatization of maintenance. Unless a target is documented (or
```

Comment:
Make this stricter and say "unless documented in `make help`"?

Reply:
Sounds good to me.